src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,31 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
+      </h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo
+          user={post.user}
+        />
```

Comment:
user can be null, we need to handle this case here

Reply:
I'm sorry, but in App.jsx, we have a function called 'getUserById.' This function checks if we have a user by their ID and returns 'null' if we can't find this user. I tried to fix your comment, but I don't understand, isn't the 'getUserById' function sufficient?